src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -90,6 +90,7 @@ internal override async Task<AuthenticationResult> ExecuteAsync(CancellationToke
 
         private async Task AcquireAuthorizationAsync(CancellationToken cancellationToken)
         {
+            AuthenticationRequestParameters.RedirectUri = _webUi?.ValidateRedirectUri(AuthenticationRequestParameters.RedirectUri);
```

Comment:
don't you risk overriding the redirect uri if the webUI is null?

Reply:
Yes, good point. In fact, we can't not have a webUI in an interactive request, so I will add a check against that. And tests around building this object with null params.